components/progress/CircularProgress.js

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import './CircularProgress.scss';
+import { classnames } from '../../helpers/component';
+
+const CircularProgress = ({
+    children,
+    className,
+    progress,
+    rootRef,
+    size = 32,
+    backgroundColor = '#555',
+    color = '#4cc75a',
+    ...rest
+}) => {
+    const backgroundCircleStyle = useMemo(
+        () => ({
+            stroke: backgroundColor
+        }),
+        backgroundColor
+    );
+
+    const circleStyle = useMemo(
```

Comment:
Since this is not really an expensive computation, I think it's fine to not use `useMemo` in this case.